pyproject.toml

```diff
@@ -2,12 +2,13 @@
 disallow_untyped_defs = true
 
 [[tool.mypy.overrides]]
-module = ['litellm', "nltk", "alignscore","pgvector.sqlalchemy", "google.auth.transport", "google.oauth2"]
+module = ['litellm', "nltk", "alignscore","pgvector.sqlalchemy", "google.auth.transport", "google.oauth2", "pydub", "vosk", "gtts"]
 ignore_missing_imports = true
 
 [tool.ruff]
 lint.select = ["E", "F", "B", "Q", "I"]
 line-length = 88
+ignore = ["B008"]  # Do not perform function calls in argument defaults.
```

Comment:
What does this do?

Reply:
```
@router.post(
    "/stt-llm-response",
    response_model=QueryResponse,
    responses={
        400: {"model": QueryResponseError, "description": "Bad Request"},
        500: {"model": AudioQueryError, "description": "Internal Server Error"},
    },
)
async def stt_llm_response(
    generate_tts: bool = Form(...),
    file: UploadFile = File(...),
    asession: AsyncSession = Depends(get_async_session),
    user_db: UserDB = Depends(authenticate_key),
) -> QueryResponse | JSONResponse:
```

in the` stt-llm-response` endpoint as we are taking a file the data is part of` multipart/form data` rather than `standard Json`, refer to [this](https://fastapi.tiangolo.com/tutorial/request-files/#__tabbed_3_1) for more clarity. so for that i had to use `file(...)` and` form(...)` as function parameters but ruff does not allow function calls in the parameters thats why i used that line to ignore that, if there are any other solution do let me know :)

Reply:
Ok this looks good!

Btw, this can be a future To-do, but have you had a chance to look at how the chat managers we want to be compatible with (Typebot, Turn.io, Glific) accepts voice files and whether this method is compatible with them? Seems like Typebot doesn't have a way to accept voice notes yet.

Reply:
I haven't looked into that yet but for sure i will do it as a future To-do, but Turn.io definitely accepts voice files

Reply:
Cool! Would you mind adding a JIRA ticket about this?